commandPrompt/PromptActual.java

```diff
@@ -0,0 +1,250 @@
+package commandPrompt;
+
+public class PromptActual {
+	public final Node ROOTNODE;
+	public Node currentNode;
+	private String currentPath;
+
+	public PromptActual() {
+		ROOTNODE = new Node("E", null, "E:\\");
+		this.currentPath = ROOTNODE.getPath();
+		currentNode = ROOTNODE;
+	}
+
+/**
+ * checking whether the command is valid or not	
+ * @param command
+ * @return
+ */
+	public boolean isValidCommand(String command) {
+		boolean flag = false;
+		String splittedCommand[] = command.split(" ");
+		if ((splittedCommand[0].equals("exit") || splittedCommand[0].equals("ls")

```

Comment:
can also check this step by creating a list of available commands and then by directly checking contains